### DISCUSSION TITLE
Number of stars on my repository goes up and down without a change in the list of people

### DISCUSSION THREAD

**Discussion by hosseinmoein** [community/community#86728] [Repositories]
### Select Topic Area

Bug

### Body

For example in this repository https://github.com/hosseinmoein/DataFrame, I have noticed that every once in a while (couple of times a month) the number of stars changes without the list of people changing

Best,
HM

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @hosseinmoein , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
